src/applications/vaos/appointment-list/components/RequestedAppointmentsList.jsx

```diff
@@ -123,6 +125,7 @@ export default function RequestedAppointmentsList({ hasTypeChanged }) {
         <div className="vads-u-background-color--gray-lightest vads-u-padding--2 vads-u-margin-y--3">
           <NoAppointments
             description="appointment requests"
+            additionalDescription={showAdditionalText}
```

Comment:
You can eliminate the boolean assignment since `<FunctionName isBoolean>` is the shorthand for `<FunctionName isBoolean={true}>`
```suggestion
            isRequest
```

You'll also need to add the `isRequest` attribute to the `<NoAppointments>` function in the `RequestedAppointmentsListGroup.jsx` file.

Reply:
fixed 👍